tests/meilisearch-test-utils.ts

```diff
@@ -1,6 +1,10 @@
-import { MeiliSearch, MeiliSearchTimeOutError, Index, sleep } from '../src/'
+import { MeiliSearch, MeiliSearchTimeOutError, Index } from '../src/'
 import { Config, EnqueuedDump } from '../src/types'
 
+async function sleep(ms: number): Promise<void> {
+  return await new Promise((resolve) => setTimeout(resolve, ms))
+}
+
```

Comment:
It was imported from the source code as a utility for the tests. I prefer having these two utilities function apart